### PR TITLE
Update ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by techtonik**
Fix lxd.log location

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ A brief description of what failed or what could be improved.
 
 # Information to attach
 
- - [ ] any relevant kernel output (dmesg)
- - [ ] container log (lxc info NAME --show-log)
- - [ ] main daemon log (/var/log/lxd.log)
+ - [ ] any relevant kernel output (`dmesg`)
+ - [ ] container log (`lxc info NAME --show-log`)
+ - [ ] main daemon log (`less /var/log/lxd/lxd.log`)
```
Comment (stgraber) [inline .github/ISSUE_TEMPLATE.md]: Probably should be "cat" rather than "less" as the user will want the whole thing in one shot.

**Comment by stgraber** [comment]
Commit is missing the Signed-off-by line.

**Comment by techtonik** [comment]
Such a pain to fix edit commits created through GitHub..

**Comment by stgraber** [comment]
Feel free to close this pull-request and send a new one. That may be the easiest way to deal with the Github UI :)

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (techtonik) [inline .github/ISSUE_TEMPLATE.md]: Done.

**Comment by techtonik** [comment]
Learn git the hard way. =)

**Pull request merged**